src/features/features.controller.ts

```diff
@@ -135,6 +242,20 @@ export class FeaturesController {
         summary:
             "Deletes a feature given the featureId (int) and user who created it is logged in.",
     })
+    @ApiResponse({
+        status: HttpStatus.NOT_FOUND,
+        description: "Feature with given ID does not exist.",
+        type: NotFoundErrorResponse,
+    })
+    @ApiResponse({
+        status: HttpStatus.UNAUTHORIZED,
+        description: "user is unauthorized to perform this action",
+        type: UnauthorizedErrorResponse,
+    })
+    @ApiResponse({
```

Comment:
Can you return the type in ApiResponse when feature is successfully deleted?

Reply:
When the feature is deleted successfully it does not return anything per the API contract, just an okay response so there's nothing to give a type to.